src/langchain_google_spanner/vector_store.py

```diff
@@ -615,18 +806,19 @@ def _validate_table_schema(self, column_type_map, types, default_columns):
                     embedding_column_type,
                 )
 
-        for column_name, column_config in column_type_map.items():
-            if column_name not in self._columns_to_insert:
-                if "NO" == column_config[2].upper():
-                    raise Exception(
-                        "Found not nullable constraint on column: {}.",
-                        column_name,
-                    )
+        if not self._skip_not_nullable_columns:
```

Comment:
We can do this validate without customer explicitly asking for it, if a column has a not null constraint and is it's not part of `_columns_to_insert ` then we should raise the exception. No matter what's the value of `_skip_not_nullable_columns `

Reply:
I've removed it and I'll keep iterating in a later PR.